grass/kernel.c

```diff
@@ -48,9 +48,19 @@ void excp_entry(uint id) {
 static void proc_yield();
 static void proc_syscall();
 
-uint proc_curr_idx;
+uint proc_curr_idx, waiting = 0, wakeup = 0;
```

Comment:
If we don't have the `sys_wait` system call, I suspect that we won't need these variables.